src/main/resources/db/migration/V0001__init.sql

```diff
@@ -0,0 +1,482 @@
+CREATE OR REPLACE VIEW sdt_user.error_messages AS SELECT * FROM sdt_owner.error_messages;
+CREATE OR REPLACE VIEW sdt_user.err_log_seq AS SELECT * FROM sdt_owner.err_log_seq;
+CREATE OR REPLACE VIEW sdt_user.err_mesg_seq AS SELECT * FROM sdt_owner.err_mesg_seq;
+CREATE OR REPLACE VIEW sdt_user.glb_par_seq AS SELECT * FROM sdt_owner.glb_par_seq;
+CREATE OR REPLACE VIEW sdt_user.global_parameters AS SELECT * FROM sdt_owner.global_parameters;
+CREATE OR REPLACE VIEW sdt_user.individual_requests AS SELECT * FROM sdt_owner.individual_requests;
+CREATE OR REPLACE VIEW sdt_user.ind_req_seq AS SELECT * FROM sdt_owner.ind_req_seq;
+CREATE OR REPLACE VIEW sdt_user.sdt_ref_seq AS SELECT * FROM sdt_owner.sdt_ref_seq;
+CREATE OR REPLACE VIEW sdt_user.service_requests AS SELECT * FROM sdt_owner.service_requests;
+CREATE OR REPLACE VIEW sdt_user.service_routings AS SELECT * FROM sdt_owner.service_routings;
+CREATE OR REPLACE VIEW sdt_user.service_types AS SELECT * FROM sdt_owner.service_types;
+CREATE OR REPLACE VIEW sdt_user.ser_rou_seq AS SELECT * FROM sdt_owner.ser_rou_seq;
+CREATE OR REPLACE VIEW sdt_user.ser_typ_seq AS SELECT * FROM sdt_owner.ser_typ_seq;
+CREATE OR REPLACE VIEW sdt_user.srv_req_seq AS SELECT * FROM sdt_owner.srv_req_seq;
+CREATE OR REPLACE VIEW sdt_user.target_applications AS SELECT * FROM sdt_owner.target_applications;
+CREATE OR REPLACE VIEW sdt_user.tar_app_seq AS SELECT * FROM sdt_owner.tar_app_seq;
+
+---------------------------------------------------------------
+-- Create Views (instead of Oracle Synonyms) for SDT_BATCH_USER
+---------------------------------------------------------------
+CREATE OR REPLACE VIEW sdt_batch_user.bulk_submissions AS SELECT * FROM sdt_owner.bulk_submissions;
+CREATE OR REPLACE VIEW sdt_batch_user.error_logs AS SELECT * FROM sdt_owner.error_logs;
+CREATE OR REPLACE VIEW sdt_batch_user.global_parameters AS SELECT * FROM sdt_owner.global_parameters;
+CREATE OR REPLACE VIEW sdt_batch_user.individual_requests AS SELECT * FROM sdt_owner.individual_requests;
+--CREATE OR REPLACE VIEW sdt_batch_user.purge AS SELECT * FROM sdt_owner.purge;
+CREATE OR REPLACE VIEW sdt_batch_user.service_requests AS SELECT * FROM sdt_owner.service_requests;
+
+
+------------------------------------------------
+-- Create Grants for SDT_OWNER
+------------------------------------------------
+
+ALTER ROLE sdt_owner SET search_path TO sdt_owner;
+ALTER ROLE sdt_user SET search_path TO sdt_user, sdt_owner;
+ALTER ROLE sdt_batch_user SET search_path TO sdt_batch_user, sdt_owner;
+
+GRANT USAGE ON SCHEMA sdt_owner TO sdt_user;
+GRANT USAGE ON SCHEMA sdt_owner TO sdt_batch_user;
+
+GRANT ALL PRIVILEGES ON ALL SEQUENCES IN SCHEMA sdt_owner TO sdt_user;
+GRANT ALL PRIVILEGES ON ALL SEQUENCES IN SCHEMA sdt_owner TO sdt_batch_user;
+
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA sdt_owner TO sdt_user;
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA sdt_owner TO sdt_batch_user;
```

Comment:
These GRANT ALL statements and the following Object Grants section are mutually exclusive.  We only want to grant specific access to specific objects for each user - which is covered by the Object Grants section - so it doesn't look like the GRANT ALL statements are needed.

Reply:
Done